File: java-version/src/builder/TestBuilder.java
package builder;

public class TestBuilder {

    public static void main(String[] args) {
        Employee employee = new EmployeeBuilder()
                .setFirstName("mohammad")
                .setLastName("alsharif")
                .setAge(24)
                .build();
        System.out.println(employee.toString());
    }
}

File: python-version/src/builder/TestBuilder.py
from src.builder.EmployeeBuilder import EmployeeBuilder

employee = EmployeeBuilder() \
    .set_employee_first_name("mohammad") \
    .set_employee_last_ame("alsharif") \
    .set_employee_age(24) \
    .build()

print(employee.firstName)
print(employee.lastName)
print(employee.age)
